low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -70,40 +70,25 @@ SSPL_LL_SETTING:
       - CPUFaultSensor
 
 RABBITMQINGRESSPROCESSOR:
-   virtual_host: SSPL
-   queue_name: actuator-req-queue
-   exchange_name: sspl-in
-   routing_key: actuator-req-key
-   username: sspluser
-   password: gAAAAABgB8tRJLZBfI6uCAtt4B8YYjveDvhHnFCbeKYsXuYxSLl3KZE5Tp94Jl2qQq9-X7x7_CinZZSh4cm9rMJ4G3HKz36I-Q==
-   primary_rabbitmq_host: localhost
+   consumer_id: sspl_actuator
+   consumer_group: cortx_monitor
+   message_type: Requests
+   offset: earliest
```

Comment:
`offset: earliest` If its earliest it will return all the messages added in queue earlier
This is now changed to `offset: latest` which only gives messages which are produced after we have started consuming messages.

Reply:
We need these in Message bus integration document, where all available options & used config is explained

Reply:
And these seems Kafka specific. Not sure if we change Msg broker later, how it would map that time, config also needs to be agnostic. @sachinpunadikar thought to consider.